draft-bbs-signatures.md

```diff
@@ -318,19 +320,25 @@ Outputs:
 
 Procedure:
 
-1. (w, h0, h) = octets_to_point(PK)
+1. (W, H0, H) = octets_to_point(PK)
 
-2. e = H(PRF(8*ceil(log2(r)))) mod r
+2. h = SHAKE256(SK  || msg[i] || ... || msg[L])
 
-3. s = H(PRF(8*ceil(log2(r)))) mod r
+3. k = h.read(64)
 
-4. b = P1 + h0 * s + h[i] * msg[i] + ... + h[n] * msg[L]
+4. e = OS2IP(k) mod q. If e = 0, go back to step 3.
 
-5. A = b * (1 / (SK + e))
+5. r = h.read(64)
 
-6. signature = (point_to_octets_min(A), e, s)
+6. s = OS2IP(r) mod q. If s = 0, go back to step 5.
 
-7. return signature
+7. b = P1 + H0 * s + H[i] * msg[i] + ... + H[n] * msg[L]
```

Comment:
`b = P1 + H0 * s + H[1] * msg[1] + ... + H[L] * msg[L]` I think, and probably `msg[1]` instead of `msg[i]` above. Should we capitalize B?